src/jsd/egd_actuator_offline.cc

```diff
@@ -51,23 +51,35 @@ void fastcat::EgdActuatorOffline::ElmoProcess()
   }
 
   // reset motor_on timer on rising edge
-  if (!last_motor_on_state_ && jsd_egd_state_.motor_on) {
+  if (!last_motor_on_state_ and jsd_egd_state_.motor_on) {
```

Comment:
For the sake of consistency I will return this back to `&&`. The rest of the code base uses `&&`.